libsharedringbuffer/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 #
-# Copyright 2022, Trustworthy Systems, UNSW
-#
+# Copyright 2022, UNSW (ABN 57 195 873 179)
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Sorry, I gave you the wrong one on the chat, it should be BSD-2-Clause for these